tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -22,15 +22,22 @@ var _ = Describe("VM backup and restore tests", Ordered, func() {
 		Expect(v).ToNot(BeNil())
 
 		if !v.IsVirtInstalled() {
-			err = v.EnsureVirtInstallation(5 * time.Minute)
+			err = v.EnsureVirtInstallation()
 			Expect(err).To(BeNil())
 			wasInstalledFromTest = true
 		}
+
+		err = v.EnsureEmulation(10 * time.Second)
+		Expect(err).To(BeNil())
+
+		err = v.EnsureDataVolume("openshift-cnv", "cirros-dv", "https://download.cirros-cloud.net/0.6.2/cirros-0.6.2-x86_64-disk.img", "128Mi", 5*time.Minute)
+		Expect(err).To(BeNil())
 	})
 
 	var _ = AfterAll(func() {
 		if v != nil && wasInstalledFromTest {
-			v.EnsureVirtRemoval(6 * time.Minute)
+			v.RemoveDataVolume("openshift-cnv", "cirros-dv", 2*time.Minute)
```

Comment:
this can be outside from if? datavolume will be always created by test?

Reply:
Okay, I moved this outside.